### PR TITLE
Sterilizine now reduces surgery step failure prob

### PR DESCRIPTION
**Pull request by Core0verload**
Finally, sterilizine has a use! When sprayed on mob with an operation started, it reduces probability of step failure by 20%. Strong alcohol drinks and miner's salve have similar effects.

This has no effect if surgery takes place on operating table and appropriate tools are used, so it's mostly useful for ghetto surgery.

Sterilizine sprayers are added to medical vendors.
medspray

This PR also refactors some of the surgery code.

:cl: CoreOverload
add: Sterilizine now reduces probability of step failure if you apply it after starting surgery. Useful while operating in less-than-perfect conditions.
add: Potent alcohol can be used as replacement for sterilizine.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
Does strong alcohol and miner salve painkillers work in spray bottles

**Comment by Core0verload** [comment]
Yes, also in patches and when splashed.

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -18,12 +18,11 @@
...
 
 /mob/living/carbon/attackby(obj/item/I, mob/user, params)
-	if(lying)
-		if(surgeries.len)
-			if(user != src && user.a_intent == "help")
-				for(var/datum/surgery/S in surgeries)
-					if(S.next_step(user, src))
-						return 1
+	if(lying && surgeries.len)
+		if(user != src && user.a_intent == "help")
```
Comment (lordpidey) [inline code/modules/mob/living/carbon/carbon_defense.dm]: couldn't we merge the if statements further?

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -39,12 +38,11 @@
 		if(D.IsSpreadByTouch())
 			ContractDisease(D)
 
-	if(lying)
+	if(lying && surgeries.len)
 		if(user.a_intent == "help")
```
Comment (lordpidey) [inline code/modules/mob/living/carbon/carbon_defense.dm]: as long as we're refactoring, we might as well fully merge the if statements.

**Comment by lordpidey** [comment]
Are the sterilizine sprays refillable?

**Pull request merged**